ui/console/src/main/scripts/plugins/metrics/plugins/metrics/ts/addUrlPage.ts

```diff
@@ -194,7 +194,8 @@ module HawkularMetrics {
           promises.push(this.HawkularMetric.AvailabilityMetricData(tenantId).query({
             availabilityId: res.id, distinct: true,
             start: 1, end: moment().valueOf()}, (resource) => {
-            res['isUp'] = (resource[0] && resource[0].value === 'up');
+              resource.reverse(); // FIXME: HAWKULAR-366
```

Comment:
This is just reversing the list not doing a real sort?  Wouldn't a real sort be better, as this is as prone to changes in the server as before? But then it is probably good enough for now.